### ISSUE TITLE
Usersnap Feedback - I'm logged in & trying to hide this spam https://participedia.net/organiza[...]

### ISSUE THREAD

**Issue by jesicarson** [participedia/usersnaps#1550]
**Sender**: pscully@clearviewconsultingllc.com
**Comment**: I'm logged in & trying to hide this spam https://participedia.net/organization/7947 -- one of 4 entries with the same title, but this message prevents me from going any further.
[Open #1231 in Usersnap Dashboard](https://usersnap.com/a/#/participedia-dev/p/participedia-net-9cb6d041/1231)

<a href='https://usersnappublic.s3.amazonaws.com/2021-12-15/18-25/1f0a5131-c934-452a-af0d-6d46a46e3d74.png'>![Usersnap Feedback - I'm logged in & trying to hide this spam https://participedia.net/organiza(...)](https://usersnappublic.s3.amazonaws.com/2021-12-15/18-25/9ce0c0aa-453e-42f0-bf6a-17ca8ffc2cdb.jpg)</a>
<a href='https://usersnappublic.s3.amazonaws.com/2021-12-15/18-25/1f0a5131-c934-452a-af0d-6d46a46e3d74.png'>Download original image</a>

**Browser**: Firefox 95 (Windows 10)
**Referer**: [https://participedia.net/organization/7947/edit](https://participedia.net/organization/7947/edit)
**Screen size**: 1760 x 1100 **Browser size**: 1760 x 985

Powered by [usersnap.com](https://usersnap.com/?utm_source=github_entry&utm_medium=web&utm_campaign=product)

--

A friendly reminder from Usersnap 👋

There’s a new widget in town. Make your bug reporting process sleeker and more advanced by changing to the new project template.

Learn more about how to transfer your project (and what new features you’ve been missing out on) in this [blog post](https://usersnap.com/blog/platform-new-widget/).

**Comment by paninee** [comment]
Hid

**Comment by plscully** [comment]
@paninee @jesicarson Thank you, Pan, for taking care of this. I now understand what I was doing wrong. I didn't understand that even users who are logged in have to use Captcha again when they want to make an edit. This is probably because it appears at the bottom of the "quick submit" form and I was working higher up in the form where Captcha was out of sight. It may just be that I am a slow learner, but it will be interesting to see if other users have this problem.

**Comment by jesicarson** [comment]
Thanks @plscully. We also noticed this issue and Pan is looking into options for the publish captcha. We may need to add captcha as an extra step that comes as a popup when you click either of the publish buttons. It's not ideal, but for now it's helping us with the spam issue. We had to add the 2nd captcha (on publish) because once spammers had created accounts they were using bots to publish. Now we're hoping this will slow them down enough to give up. Stay tuned!
